app/views/teacher_interface/application_forms/show.html.erb

```diff
@@ -65,6 +65,10 @@
 <% elsif @view_object.application_form.declined? %>
   <h2 class="govuk-heading-l">Your QTS application has been declined</h2>
 
+  <%- if @view_object.show_further_information_request_expired_content? -%>
+    <%= govuk_inset_text(text:t("assessor_interface.assessment_sections.show.failure_reasons.further_information_request_expired")) %>
```

Comment:
Since this key will never be selected by an assessor, or set in the `failure_reason`, I wonder if this string belongs somewhere else, specifically we could do something like:

```rb
t(".further_information_request_expired")
```

And then set the key to: `assessor_interface.application_forms.show.further_information_request_expired `.

Reply:
I wondered that too. If we do that then the possible failure reasons are now in two places which is potentially confusing too. We might have a need to store them somewhere else altogether as they're not presenting very well in the analytics. Let's discuss this later.

Reply:
Yeah, happy to discuss it.

I think in this case I'd argue that this isn't a "failure reason" as we've defined them, as they're specifically related to the initial assessment stage and associated with assessment sections, rather than further information requests. The `failure_reason_key` on the further information request comes directly from the assessment section after all.

This is more of a "further information request declined reason", i.e. it's the `failure_reason_assessor_feedback` equivalent for an FI request which we currently don't have a concept of.